manifests/0000_70_cluster-network-operator_03_daemonset.yaml

```diff
@@ -16,7 +16,7 @@ spec:
     spec:
       containers:
       - name: network-operator
-        image: docker.io/openshift/origin-cluster-network-operator:v4.0.0
+        image: quay.io/openshift/origin-cluster-network-operator:4.2
```

Comment:
Should this be ":latest" until the the 4.2 branch is made?

Reply:
Good question. The latest tag is unstable, and moves around between releases (probably a bug, but not going to worry about that right now). The mirroring destination is independent of the branch name. So this is what we got.